src/table/table.tsx

```diff
@@ -10,6 +10,7 @@ import { PaginateEvent, TableTop } from './table-top';
 interface TableAction<T> {
     id: string;
     hidden?: boolean | ((row: T) => boolean);
+    qa?: string | ((row: T, index: number) => string);
```

Comment:
I find just QA on an API of a table action ab bit short. Maybe sth like `QaTaggedBy`. Sth that allows strings and functions while still giving a bit more info.

Reply:
Took the lead from the hidden callback one line above :D But yeah i know what you mean. How about `renderQaTag`?